src/plugins/plot/stackedPlot/StackedPlotItem.vue

```diff
@@ -145,19 +187,73 @@ export default {
             this.status = status;
             this.updateComponentProp('status', status);
         },
+        setSelection() {
+            let childContext = {};
+            childContext.item = this.object;
+            this.context = childContext;
+            if (this.removeSelectable) {
+                this.removeSelectable();
+            }
+
+            this.removeSelectable = this.openmct.selection.selectable(
+                this.$el, this.context);
+        },
         loadingUpdated(loaded) {
             this.loading = loaded;
             this.updateComponentProp('loading', loaded);
         },
         getProps() {
             return {
+                limitLineLabels: this.showLimitLineLabels,
                 gridLines: this.gridLines,
                 cursorGuide: this.cursorGuide,
                 plotTickWidth: this.plotTickWidth,
                 loading: this.loading,
                 options: this.options,
-                status: this.status
+                status: this.status,
+                colorPalette: this.colorPalette
             };
+        },
+        getPlotObject() {
+            if (this.object.configuration && this.object.configuration.series) {
+                //If the object can be persisted, allow initialization of the config from it's persisted config
```

Comment:
I guess the logic specifically here is whether or not it has stored configuration, not whether or not it can be persisted. There can (and will) be non-persistable plots with configuration (eg. served from static JSON). I think the logic is correct, the comment just needs to be aligned with the logic.